### PR TITLE
Js feature

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone I

In this PR we did the next:

- Create `bookList` array that keeps a list of books.
- Create `addBook` function to add a new book to the array, with title and author.
- Create `removeBook` function to remove a book from the array.
- Display all books saved in the array in the top part of the page with the `drawBooks` function.
- When a user clicks the `Add` button:
  - A new book is added to the collection.
  - The new book is displayed on the page.
- When a user clicks the `Remove` button:
  - The correct book is removed from the collection.
  - The correct book disappears from the page.
- Books data is preserved in the browser's memory by using `localStorage`.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+if (stringBookList) {
+  const parsedBookList = JSON.parse(stringBookList);
+  if (Array.isArray(parsedBookList)) {
+    bookList = parsedBookList;
+  }
+}
+
+function removeBook(index) {
+  bookList.splice(index, 1);
+//   drawBooks();
```
Comment (TedLivist) [inline js/main.js]: - [ ] Kindly remove commented code.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @anthonymr and @lilskyex0x,

Good job so far!
✔️ Books are loaded from the storage
✔️ Books are stored in the localStorage
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (anthonymr) [reply js/main.js]: Thanks very much, we forgot to remove that line.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @anthonymr and @lilskyex0x,

✔️ Clean code

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**